### PR TITLE
MTDSA-923 Remove UK property accounting type, AND removed the GET end…

### PR DESCRIPTION
**Pull request by alexec**
…point as UK property does not have any fields anymore

### CODE REVIEW

**Comment by GlassAndOneHalf** [comment]
The create operation should be turned into a POST with no content. There is no need for a model class anymore.

**Comment by alexec** [comment]
I wanted to minimise breaking change.  Do you think it should accept any body (Postel law)? Or reject anything that is not an object?

File: test/uk/gov/hmrc/selfassessmentapi/resources/Jsons.scala [hunk 1]
```diff
@@ -103,9 +103,7 @@ object Jsons {
   }
 
   object Properties {
-    def apply(accountingType: String = "CASH"): JsValue = {
-      Json.parse("{}".stripMargin)
-    }
+    def apply(): JsValue = Json.parse("{}")
```
Comment (GlassAndOneHalf) [inline test/uk/gov/hmrc/selfassessmentapi/resources/Jsons.scala]: You can use `Json.obj()` here.

**Comment by alexec** [comment]
Can this be merged?

**Pull request merged**